pkg/storageincentives/redistribution/inclusionproof.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Used for inclusion proof utilities
```

Comment:
This comment doesn't add much value, make the description more detailed or remove the comment.